CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/NoResult/NoResultScreenViewController.swift

```diff
@@ -88,13 +88,21 @@ final public class NoResultScreenViewController: UIViewController {
         }
         fatalError("No result header not found")
     }()
-
     private (set) var dataSource: HelpDataSource
     private var giniConfiguration: GiniConfiguration
     private let tableRowHeight: CGFloat = 44
     private let sectionHeight: CGFloat = 70
     private let type: NoResultType
     private let viewModel: NoResultScreenViewModel
+    private var buttonsHeightConstraint: NSLayoutConstraint?
+    private var numberOfButtons: Int {
+        return [
+            viewModel.isEnterManuallyHidden() == false,
```

Comment:
[Suggestion] viewModel.isEnterManuallyHidden() is already a Boolean, it should not be compared to false.